### PR TITLE
Add missing presenters unit tests

### PR DESCRIPTION
**Pull request by pawelpaszki**
## Description
Add missing presenters unit tests

## Checklist (Definition of Done)

- [x] All acceptance criteria specified in JIRA have been completed
- [x] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- ~~[ ] Documentation added for the feature~~
- [x] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- ~~[ ] Required metrics/dashboards/alerts have been added (or PR created).~~
- ~~[ ] Required Standard Operating Procedure (SOP) is added.~~
- ~~[ ] JIRA has created for changes required on the client side~~

### CODE REVIEW

File: internal/kafka/internal/presenters/admin_kafka_test.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			converted, err := PresentKafkaRequestAdminEndpoint(tt.args.dbKafkaRequest, tt.args.accountService)
+			if !tt.wantErr && err != nil {
+				t.Errorf("unexpected error for PresentKafkaRequestAdminEndpoint: %v", err)
+			}
+			if !reflect.DeepEqual(converted, tt.want) {
```
Comment (ziccardi) [inline internal/kafka/internal/presenters/admin_kafka_test.go:90]: Can we use `gomega` and change all these `DeepEqual` checks to something like
```
Expect(converted).To(Equal(tt.want)
```
? (`Equal` internally uses `reflectDeepEqual`)

Comment (machi1990) [reply internal/kafka/internal/presenters/admin_kafka_test.go:90]: Very good point. I think we should have this suggested as a best practice / team testing style and document it.

Comment (pawelpaszki) [reply internal/kafka/internal/presenters/admin_kafka_test.go]: @machi1990 @ziccardi - I have addressed your comments. do you mind taking another look?

File: internal/kafka/internal/presenters/metric.go [hunk 2]
```diff
@@ -88,16 +88,15 @@ func PresentMetricsByInstantQuery(metrics *observatorium.KafkaMetrics) ([]public
...
+		if label == labelName {
 			return true
 		}
 	}
-
 	return false
 }
 
-func getSupportedLables() []string {
+func getSupportedLabels() []string {
```
Comment (machi1990) [inline internal/kafka/internal/presenters/metric.go:100]: good catch!

File: internal/kafka/internal/presenters/metric.go [hunk 3]
```diff
@@ -88,16 +88,15 @@ func PresentMetricsByInstantQuery(metrics *observatorium.KafkaMetrics) ([]public
 	return out, nil
 }
 
-func isAllowedLabel(lable string) bool {
-	for _, labelName := range getSupportedLables() {
-		if lable == labelName {
+func isAllowedLabel(label string) bool {
```
Comment (machi1990) [inline internal/kafka/internal/presenters/metric.go:91]: good catch!

File: internal/kafka/internal/presenters/metric_test.go [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+			if !tt.wantErr && err != nil {
+				t.Errorf("unexpected error for PresentMetricsByRangeQuery: %v", err)
+			}
+			if !tt.wantErr {
+				firstWantQuery := tt.want[0]
+				if firstQuery[0].Metric["__name__"] != firstWantQuery.Metric["__name__"] {
+					t.Errorf("unexpected result for PresentMetricsByRangeQuery - want: %+v got: %+v", tt.want, firstQuery)
+				}
+			}
+		})
```
Comment (machi1990) [inline internal/kafka/internal/presenters/metric_test.go:209]: We can use `gomega.Expect...` for these too

File: internal/kafka/internal/presenters/metric_test.go [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+
+	RegisterTestingT(t)
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			query, err := PresentMetricsByInstantQuery(tt.args.metrics)
+			if !tt.wantErr && err != nil {
+				t.Errorf("unexpected error for PresentMetricsByInstantQuery: %v", err)
+				return
+			}
```
Comment (machi1990) [inline internal/kafka/internal/presenters/metric_test.go:249]:
```suggestion
			if !tt.wantErr && err != nil {
				t.Errorf("unexpected error for PresentMetricsByInstantQuery: %v", err)
				return
			}
			Expect(tt.wantErr).To(Equal(err != nil), "unexpected error for PresentMetricsByInstantQuery")
```

File: internal/kafka/internal/presenters/object_reference.go [hunk 6]
```diff
@@ -36,6 +39,8 @@ func objectKind(i interface{}) string {
 		return KindCloudProvider
 	case errors.ServiceError, *errors.ServiceError:
 		return KindError
+	case api.ServiceAccount, *api.ServiceAccount:
+		return KindServiceAccount
```
Comment (machi1990) [inline internal/kafka/internal/presenters/object_reference.go:43]: Good catch!

**Review by machi1990 — APPROVED** [review]
lgtm

left a general comment about the usage of gomega.Expect that I feel like it can be used but this shouldn't be a blocker! 

Thanks for the great work @pawelpaszki

File: internal/kafka/internal/presenters/metric_test.go (hunk 4 above)
Comment (pawelpaszki) [reply internal/kafka/internal/presenters/metric_test.go]: thanks Manyanda. I have fixed that too

**Pull request merged**